src/graphql.js

```diff
@@ -19,23 +18,9 @@ import {
 } from './script';
 import { buildPayPalUrl } from './domains';
 
-
-type FundingEligibilityParams = {|
```

Comment:
@borodovisin this is an interesting strategy for fixing the type errors 😺 Deleting types and using mixed technically works...

Reply:
Do we expose this type anywhere else? I think we might if so we can just use that. Otherwise I think I'm okay with mixed for the short term. I feel like this should be a type we import in the future.